ObjectDetection/ObjectDetection.xcodeproj/project.pbxproj

```diff
@@ -26,9 +26,11 @@
 		269E7498255CCBA000B1D6CA /* InferenceModule.mm in Sources */ = {isa = PBXBuildFile; fileRef = 269E7496255CCBA000B1D6CA /* InferenceModule.mm */; };
 		269E749B255CCBC300B1D6CA /* UIImage+Helper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 269E749A255CCBC300B1D6CA /* UIImage+Helper.swift */; };
 		26FAB5692565E8FC00EA6ED6 /* PrePostProcessor.swift in Sources */ = {isa = PBXBuildFile; fileRef = 26FAB5682565E8FC00EA6ED6 /* PrePostProcessor.swift */; };
+		E59C7A0EE37EBF9D5E6472AB /* Pods_ObjectDetection.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = E1FB1B65020C9B26673B0368 /* Pods_ObjectDetection.framework */; };
```

Comment:
Could you remove all pod related changes to the project file? You can do that by `pod deintegrate`.

Reply:
Thanks, sure. The PR is still in draft mode and I'll add the script and do some cleanup before setting it to be Ready. Btw, a draft PR that gets approved won't be merged to master, right?